pdpserver/server/log_mgr.go

```diff
@@ -0,0 +1,19 @@
+package server
```

Comment:
Please rename log_mgr.go to logger.go. There is no management just additional logger interface.